wat2wasm/src/main/java/com/dylibso/chicory/wat2wasm/Wat2Wasm.java

```diff
@@ -0,0 +1,73 @@
+package com.dylibso.chicory.wat2wasm;
+
+import static java.nio.file.Files.copy;
+
+import com.dylibso.chicory.log.Logger;
+import com.dylibso.chicory.log.SystemLogger;
+import com.dylibso.chicory.runtime.HostImports;
+import com.dylibso.chicory.runtime.Module;
+import com.dylibso.chicory.runtime.exceptions.WASMMachineException;
+import com.dylibso.chicory.wasi.WasiExitException;
+import com.dylibso.chicory.wasi.WasiOptions;
+import com.dylibso.chicory.wasi.WasiPreview1;
+import com.dylibso.chicory.wasm.exceptions.MalformedException;
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.FileSystem;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.List;
+
+public class Wat2Wasm {
+    private static final Logger logger = new SystemLogger();
+
+    public static byte[] parse(File file) {
+        var module = Module.builder("wat2wasm").build();
```

Comment:
Would it be possible to memoize this? it shouldn't change over the execution of the program.